.prettierignore

```diff
@@ -9,6 +9,7 @@
 babel.config.js
 postcss.config.js
 
+/app/webpacker/css/admin_v3/globals/
```

Comment:
I'm wondering why you've added the file here? You're happy with the existing indentation and prettier will erase that?

Reply:
Can we ignore that particular file? Instead of ignoring the whole folder?

Reply:
Good point, thanks I've updated this!